fenix/app/src/main/res/values-night/colors.xml

```diff
@@ -9,7 +9,7 @@
     <!-- Default Screen background, Frontlayer background, App Bar Top, App Bar Bottom, Frontlayer header -->
     <color name="fx_mobile_layer_color_1">@color/photonDarkGrey60</color>
     <!-- Card background, Menu background, Dialog, Banner -->
-    <color name="fx_mobile_layer_color_2">@color/photonDarkGrey30</color>
+    <color name="fx_mobile_layer_color_2">@color/photonDarkGrey70</color>
```

Comment:
Seems like this is the only change proposed to fix the issue?
Given that this is used in many places all throughout the app the change will have to be approved by UX. Will keep you posted with the discussions result.

Reply:
These values are provided by UX design system. So, this shouldn't be changed externally.